src/analytics/analytics-tracker-window.ts

```diff
@@ -1,9 +1,67 @@
+import { isAnalyticsTrackerWindow } from './is-analytics-step-tracker-window';
+
 export interface AnalyticsTracker {
     track(step: string, data: any): void;
-    hit(hitName: string, data: any): void;
-    hasPayloadLimit(data: any): boolean;
 }
 
 export default interface AnalyticsTrackerWindow extends Window {
     analytics: AnalyticsTracker;
 }
+
+interface AnalyticsTrackerWindowGA extends AnalyticsTrackerWindow {
+    ga(command: string, eventName: string, payload: AnalyticPayload): void;
+}
+
+function isAnalyticsTrackerWindowGA(window: Window | AnalyticsTrackerWindowGA): window is AnalyticsTrackerWindowGA {
+    return window && 'ga' in window && typeof window.ga === 'function';
+}
+
+export const isGoogleAnalyticsAvailable = () => {
+    return isAnalyticsTrackerWindow(window) && isAnalyticsTrackerWindowGA(window);
+};
+
+export const sendGoogleAnalytics = (type: string, payload: AnalyticPayload) => {
```

Comment:
All these exported functions should be placed in their own files rather than under `analytics-tracker-window`, which is only ever intended to contain `AnalyticsTrackerWindow` as the name of the file suggests.

Reply:
@davidchin done!